compiler/test/dotty/tools/dotc/printing/SyntaxHighlightingTests.scala

```diff
@@ -89,12 +111,12 @@ class SyntaxHighlightingTests {
 
   @Test
   def unionTypes = {
```

Comment:
Let remove this test and move it to `def types`. Also I don't think we need all these combinations. Let's keep:
```scala
test("type A = String|Int", "<K|type> <T|A> = <T|String|Int>")
test("type B = String & Int", "<K|type> <T|A> = <T|String & Int>")
test("fn[String](input)", "fn[<T|String>](input)")
```